src/writer/titleListingHandler.cpp

```diff
@@ -88,19 +77,13 @@ void TitleListingHandler::stop() {
 
 DirentHandler::Dirents TitleListingHandler::createDirents() const {
   Dirents ret;
-  ret.push_back(mp_creatorData->createDirent(NS::X, "listing/titleOrdered/v0", "application/octet-stream+zimlisting", ""));
-  if (m_hasFrontArticles) {
```

Comment:
`TitleListingHandler::m_hasFrontArticles` can be dropped.

I propose to split the first commit as follows:

1. Make `listing/titleOrdered/v1` be created unconditionally. This is a small code change but a significant functional change on its own that takes about half of the description of the current version of the first commit in the PR. Removal of `TitleListingHandler::m_hasFrontArticles` belongs here.
2. Retirement of `listing/titleOrdered/v0`

Reply:
Done